Appium/generate_config.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+if [ -z "$PLATFORM"]; then
+  PLATFORM="Android"
```

Comment:
Removed it from config, but forgot to remove the definition of parameter. Will fix it.